person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  @obj_count = 0
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @obj_count += 1
+    @id = @obj_count
```

Comment:
- [ Optional ] Great job. the way you generate and increment the `@id` is ok but not the best because there is a very high probability of having many books with the same id. Ill recommend using the `Random` method that auto-generates an ID in a specific range
i.e `@id = Random.rand(1..1000)`